Gemfile.jruby-1.9.lock

```diff
@@ -245,10 +245,10 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       nokogiri
       xml-simple
-    logstash-input-beats (3.1.4)
+    logstash-input-beats (3.1.4-java)
       concurrent-ruby (>= 0.9.2, <= 1.0.0)
       jar-dependencies (~> 0.3.4)
-      logstash-codec-multiline (~> 3.0)
+      logstash-codec-multiline (>= 2.0.5)
```

Comment:
ah, https://github.com/logstash-plugins/logstash-input-beats/commit/1a0b0b217ac98595fee28a4b2f3a34437e763e5e

LS 2.4 and the plugin api 👍